test/setup_logging_test.py

```diff
@@ -0,0 +1,173 @@
+from luigi.setup_logging import DaemonLogging, InterfaceLogging
+from luigi.configuration import LuigiTomlParser, get_config
+from helpers import unittest
+
+
+class TestDaemonLogging(unittest.TestCase):
+    cls = DaemonLogging
+
+    def setUp(self):
+        self.cls.configured = False
+
+    def tearDown(self):
+        self.cls.configured = False
+        self.cls.config = get_config()
+
+    def test_cli(self):
+        opts = type('opts', (), {})
+
+        opts.background = True
+        result = self.cls._cli(opts)
+        self.assertTrue(result)
+
+        opts.background = False
+        opts.logdir = './tests/'
+        result = self.cls._cli(opts)
+        self.assertTrue(result)
+
+        opts.background = False
+        opts.logdir = False
+        result = self.cls._cli(opts)
+        self.assertFalse(result)
+
+    def test_section(self):
```

Comment:
This test name `setup_logging_test.TestDaemonLogging.test_section` could have more context if you rename `test_section` to maybe `test_section_returns_true_with_config` and another test `test_section_returns_false_without_config`.

I know we don't follow this style at most places in this code base. But code testing improvements can start somewhere. If you want you can do this is too much you can do this as a follow up.

Reply:
It's test all what we need test for `_section` method

Reply:
The point is that `test_section` is not saying much compared to `test_section_returns_true_with_config`.